lib/pages/authentication-page.dart

```diff
@@ -0,0 +1,261 @@
+import 'package:MobileOne/localization/localization.dart';
+import 'package:MobileOne/services/user_service.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:MobileOne/services/authentication_service.dart';
+import '../localization/localization.dart';
+
+class AuthenticationPage extends StatefulWidget {
+  AuthenticationPage({Key key}) : super(key: key);
+
+  @override
+  AuthenticationPageState createState() => AuthenticationPageState();
+}
+
+class AuthenticationPageState extends State<AuthenticationPage> {
+  final _formKey = GlobalKey<FormState>();
+  final _emailController = new TextEditingController();
+  final _passwordController = new TextEditingController();
+  String _email;
+  String _password;
+
+  @override
+  void dispose() {
+    _emailController.dispose();
+    _passwordController.dispose();
+    super.dispose();
+  }
+
+  void handleSubmittedPassword(String input) {
+    _password = input;
+  }
+
+  void handleSubmittedEmail(String input) {
+    _email = input;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      body: Center(
```

Comment:
wrap Center widget with a SafeArea to avoid content under the notification bar of the phone